syncer/proto/syncer.proto

```diff
@@ -0,0 +1,27 @@
+syntax = "proto3";
+
+package v1;
+
+option go_package = "/syncer/proto";
+
+import "google/protobuf/empty.proto";
+
+service SyncPeer {
+  rpc GetBlocks(GetBlocksRequest) returns (stream Block);
+  rpc GetStatus(google.protobuf.Empty) returns (SyncPeerStatus);
+}
+
+message GetBlocksRequest {
+  // The beginning height
+  uint64 from = 1;
+}
+
+message Block {
+  // RLP Encoded Block Data
+  bytes block = 1;
+}
+
+message SyncPeerStatus {
+  // Latest block height
+  uint64 number = 1;
+}
```

Comment:
Please add more in-code docs for these proto concepts 🙏

Reply:
ping @Kourin1996

Reply:
@zivkovicmilos 3a189f27dc13901076086fd4eb4b49bb1ef153b6